src/features/offer/components/OfferTile/OfferTile.tsx

```diff
@@ -84,7 +75,7 @@ const UnmemoizedOfferTile = (props: OfferTileProps) => {
     <View {...getHeadingAttrs(3)}>
       <StyledTouchableLink
         highlight
-        height={height + MAX_OFFER_CAPTION_HEIGHT}
+        height={height + theme.tiles.maxCaptionHeight.newOfferTile}
```

Comment:
Il faudrait renommer `newOfferTile` en `offerTile` dans le theme
```suggestion
        height={height + theme.tiles.maxCaptionHeight.offerTile}
```

Reply:
Bien vu!